### PR TITLE
chore: release kong mesh 1.8.0 and 1.7.1

### PR DESCRIPTION
**Pull request by bartsmykla**
### Summary
Prepare `release/mesh-1.8` branch for the real release with changelog updated etc.

- [x] ~~blocked by: https://github.com/kumahq/kuma-website/pull/837~~

### Reason
n/a

### Testing

n/a

### CODE REVIEW

File: app/mesh/changelog.md [hunk 1]
```diff
@@ -4,6 +4,52 @@ no_search: true
...
+Built on top of [Kuma 1.7.0](https://github.com/kumahq/kuma/releases/tag/1.7.0)
+
+### Changes
+
+New Features:
+
+- Support for arm64
+- Graceful shutdown of OPA
+- Role-based AWS auth for Vault
+- Add Vault AWS Auth option to set Server ID
```
Comment (lena-larionova) [inline app/mesh/changelog.md:20]:
```suggestion
- Role-based AWS authentication for Vault
- Added a Vault AWS authentication option to set the server ID
```
What is the option? Can you add it as: "Added the `<option-name>` Vault AWS authentication option to set the server ID."

File: app/mesh/changelog.md [hunk 2]
```diff
@@ -4,6 +4,52 @@ no_search: true
...
+- Bump github.com/hashicorp/go-retryablehttp from 0.6.6 to 0.7.1
+- Bump github.com/open-policy-agent/opa from 0.38.1 to 0.40.0
+- Bump github.com/open-policy-agent/opa-envoy-plugin from 0.38.1-envoy-3 to 0.40.0-envoy
+- Bump k8s.io/api from 0.23.6 to 0.24.1
+- Bump k8s.io/apimachinery from 0.23.6 to 0.24.1
+- Bump sigs.k8s.io/controller-runtime from 0.11.2 to 0.12.1
+
+### Upgrading
+
+### kubectl
```
Comment (lena-larionova) [inline app/mesh/changelog.md:34]:
```suggestion
#### kubectl
```

File: app/mesh/changelog.md (hunk 1 above)
Comment (bartsmykla) [reply app/mesh/changelog.md]: I believe the option is called `Server ID` :-)

Comment (bartsmykla) [reply app/mesh/changelog.md]: > Added a Vault AWS authentication option to set the server ID header

Do you think it's ok now?

**Review by lena-larionova — APPROVED** [review]
LGTM, just needs the release date. Installer script works also.

**Pull request merged**